public/index.html

```diff
@@ -2,14 +2,37 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+   
+    <!-- Fab icon  -->
+    <link rel="icon" type="image/x-icon" href="favicon.ico">
+   
+    
+
+    <!-- Title & Discription Tags -->
+    <meta name="title" content="First Step Farma">
+    <meta name="description" content="Some description, I will write later">
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://gantt-demo-3c430.web.app/">
+    <meta property="og:title" content="Gantt Chart — Timeline">
+    <meta property="og:description" content="Some description, I will write later">
+    <meta property="og:image" itemprop="image" content="/assets/images/thumbnail.png">
+
+    <!-- Twitter -->
+    <meta property="twitter:card" content="summary_large_image">
+    <meta property="twitter:url" content="https://gantt-demo-3c430.web.app/">
+    <meta property="twitter:title" content="Gantt Chart — Timeline">
+    <meta property="twitter:description" content="Some description, I will write later">
+    <meta property="twitter:image" content="/assets/images/thumbnail.png">
+
+    <!-- Theme Color tag  -->
+    <meta name="theme-color" content="#000000" />
```

Comment:
Colour value should be the green colour in our icon which is the same as our primary colour.